NGUIButtonInsetsExample/ViewController.swift

```diff
@@ -202,26 +215,26 @@ final class AllInsetsView: UIStackView {
         super.init(frame: .zero)
         self.axis = .vertical
         self.spacing = 30
-
+        
```

Comment:
Sorry, to be clear, there are a bunch of lines in this file which have trailing whitespace (including this one).  I think you should be able to check them out by looking at the github diff. We should remove the trailing whitespace from all of these 👍

Reply:
Oh, I see, sure

Reply:
I have a new Mac, so I forgot that setting is not the default. I set it up now.